### PR TITLE
Update default values 

### PR DESCRIPTION
**Pull request by andreeaceachir142**

### CODE REVIEW

File: packages/map-template/.github/README.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ The Map Template supports using query parameters for all the properties provided
 
 The supported query parameters are the following:
 
-1. `apiKey` - Used like this `apiKey=yourApiKey`. If no apiKey is provided, the app will default to `3ddemo`.
+1. `apiKey` - Used like this `apiKey=yourApiKey`. If no apiKey is provided, the app will default to `02c329e6777d431a88480a09`.
```
Comment (ammapspeople) [inline packages/map-template/.github/README.md:91]: Do we have a more "readable" alias for that solution?

Comment (andreeaceachir142) [reply packages/map-template/.github/README.md:91]: We have an alias `mapspeople3d` as well

Comment (andreeaceachir142) [reply packages/map-template/.github/README.md]: 🫡

**Review by ammapspeople — APPROVED** [review]
✅

**Pull request merged**